docs/rpyc.rst

```diff
@@ -30,9 +40,18 @@ use RPyC with ev3dev:
 
       chmod +x rpyc_server.sh
 
-   Launch the created file either from ssh session, or from brickman.
+   Launch the created file either from ssh session (with ``./rpyc_server.sh``
```

Comment:
Can you link to the SSH tutorial here for those who haven't already configured it?